catch: 
    const error = new CustomError(err.status, err.message)
        res.status(error.status).json({ status: error.status, error: error.message, data: null })
    
200:
    res.status(200).json({ status: 200, error: null, data:  })